hummingbird/ml/supported.py

```diff
@@ -45,11 +46,12 @@ def _build_sklearn_operator_list():
             ExtraTreesClassifier,
             ExtraTreesRegressor,
         )
-        from sklearn.tree import DecisionTreeClassifier
+        from sklearn.tree import DecisionTreeClassifier, DecisionTreeRegressor
 
         return [
             # Tree-methods
             DecisionTreeClassifier,
+            DecisionTreeRegressor,
             RandomForestClassifier,
             RandomForestRegressor,
             GradientBoostingClassifier,
```

Comment:
Same here. I feel we should reorder alphabetically for ease of reading. WDYT?

Reply:
Agree.